docs/releasing-files.md

```diff
@@ -165,16 +165,37 @@ If you are likely to release data multiple times, e.g. for initial discussion wi
+Having applied disclosure controls to the aggregated study data, you are ready to request their release. **Only specific members of the OpenSAFELY team trained in output checking have permissions to release the data**. When you are ready to request a release of your aggregated results, and only the minimum outputs necessary please [complete this form](documents/OpenSAFELY_Output_Review_Form_07-04-22.docx). For each output wishing to be released you will need to provide a clear contextual description including:
 
 1. Variable descriptions
 2. A description and count of the underlying population for each output.
 3. Relationship to other data/tables which through combination may introduce secondary disclosive risks.
 
-!!! note
-    Only the following file types will be reviewed: HTML; TXT; CSV; TSV; SVG; JPG; JSON; PNG. (Email datarelease@opensafely.org if you need additional file types).
+### Allowed file types
+
+Only certain file types will be reviewed and released from the secure server. See below for details on each type:
+
+* **Tables** - Tables should be produced as either `csv` or `tsv` files. 
+    * Make sure that any column names are understandable for reviewers.
+    * Limit the number of columns or rows to only what is necessary.
+* **Figures** - Figures can be produced as bitmap images (`jpeg` or `png`) or vector graphics (`svg`).
+    * Underlying data for all figures should be provided for review (even if it is not being released).
+    * Consider requesting release of the underlying data if you think you will need to make stylistic changes to the figures so that you don't need to request multiple reviews.
+    * If you are producing a lot of figures, consider combining them into panel plots or requesting release of the underlying data so that you can produce them outside of the secure server.
+* **Other**
+    * `txt` files can be released, but you should consider whether the output can be produced as a table, which is easier to review.
+    * `json` files can be released, but as with tables, make sure that the attributes are easily understandable for reviewers. If the output can be represented as a table, you should consider converting it.
+    * `html` files can be released if you are producing a report style output, but please note the points below:
+        * `html` files are harder to review than other output types, so should be reserved for reports which require both contextual text and embedded outputs. Most commonly, this will be a report to be hosted on [OpenSAFELY Reports](https://reports.opensafely.org/) (more information [here](https://docs.opensafely.org/reports/intro/)). If you can produce your report locally, using individually released files, you should.
+        * Hide any code blocks from the report. 
```

Comment:
is "Hide" the right term here? "Remove" or is "Hide" the terminology used by  rmd/jupyter  etc?

Reply:
I think you're right "hide" isn't quite right here, but we don't want people to actually "remove" code blocks. I'll change this to something along the lines of "Make sure the code blocks aren't rendered in the final html if you don't need them. You can do this by...."